mkosi/config.py

```diff
@@ -4562,6 +4562,10 @@ def parse_config(
             continue
 
         if v_i > 0 and argv[v_i - 1] != "--":
+            # If -- is already in the command line after the verb, we have to remove it.
+            if "--" in argv[v_i:]:
+                del argv[v_i + argv[v_i:].index("--")]
+
```

Comment:
I'm not sure I understand the semantics here. E.g. for `journalctl` we document since yesterday
> All arguments specified after the `journalctl` verb and separated by `--` from the regular options are appended to the **journalctl** invocation.

but with this change we'd effectively always move the `--` before the verb:
```
>>> argv = ["mkosi", "--mkosi1", "journalctl", "--mkosi2", "--", "--journalctl1"]
>>> v_i = argv.index("journalctl")
>>> del argv[v_i + argv[v_i:].index("--")]
>>> argv
['mkosi', '--mkosi1', 'journalctl', '--mkosi2', '--journalctl1']
>>> argv.insert(v_i, "--")
>>> argv
['mkosi', '--mkosi1', '--', 'journalctl', '--mkosi2', '--journalctl1']
```